docs/syntax_and_semantics/instance_alignof.md

```diff
@@ -0,0 +1,18 @@
+# instance_alignof
+
+The `instance_alignof` expression returns an `Int32` with the instance alignment of a given class. For example:
```

Comment:
**chore:** Add a reference to `alignof`.
```suggestion
The `instance_alignof` expression returns an `Int32` with the instance alignment of a given class. It's the equivalent to `[alignof](alignof.md)` but for reference types, where `alignof` refers to the alignment of the pointer representation.
For example:
```

Reply:
The wording sounds rather confusing. If it is already "equivalent" to `alignof` what difference does it make?

Reply:
Feel free to adjust the wording to make more sense for you.